qiskit/chemistry/components/variational_forms/chc.py

```diff
@@ -0,0 +1,194 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Compact heuristic ansatz for Chemistry """
+
+from typing import List, Optional, Union
+
+import numpy as np
+
+from qiskit import QuantumRegister, QuantumCircuit
+
+from qiskit.circuit import ParameterVector, Parameter
+
+from qiskit.aqua.components.variational_forms import VariationalForm
+from qiskit.aqua.components.initial_states import InitialState
+
+
+class CHC(VariationalForm):
+    """ This trial wavefunction is the Compact Heuristic for Chemistry.
+
+    The trial wavefunction is as defined in
+    Ollitrault Pauline J., Chemical science 11 (2020): 6842-6855. It aims at approximating
+    the UCC Ansatz for a lower CNOT count.
+
+    Note:
+        It is not particle number conserving and the accuracy of the approximation decreases
+        with the number of excitations.
+    """
+
+    def __init__(self, num_qubits: int, depth: int = 1, ladder: bool = False,
```

Comment:
While we used to call the number of repetitions depth this has been altered and we now call that reps - so this param should be renamed.

Reply:
Updated in da221021bab30929d64f160b5db54d83f087c918